scala/src/main/scala/smile/clustering/distance/DistanceUtils.scala

```diff
@@ -0,0 +1,22 @@
+package smile.math.distance
+
+/**
+ * @author Beck Gaël
+ *
+ **/
+object DistanceUtils
```

Comment:
Who uses this class?

Reply:
It's an utility to choose the distance but yes, user can make it by himself